charters/DataOperations/charter.md

```diff
@@ -0,0 +1,101 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team (DataOps) is a user-focused group aiming to connect the scientific community to the DCP through the effective tools and resources generated by the DCP which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution. These products include data, metadata, analysis tools, data access features. DataOps ensures that data flows through the DCP end-to-end in an efficient manner, is appropriately validated, and fulfills the needs and expectations of data consumers. DataOps maintains a DCP-wide vision through their internal focus on end-to-end data flow, as well as their community-focused goal of conveying the value of the entire DCP.
```

Comment:
This description is a little different than I expected.  There are certain elements here that feel weird to not be the job of all components in the DCP. For instance, I would imagine it is all of the components areas of focus to "identifying the value-added for the HCA consumers" as part of the development process, why would another group do that for each team? Additionally, user value definition, value management, and measurement (through strategic aims metrics) is specifically called out as part of theme ownership responsibility. I would also hope we are all focused on delivering "useful, fully vetted products" at the component level and between components. There are other examples here in this paragraph that also seem odd to be the purview of a specific team, I am happy to list more if needed. I wonder why the major focus/framing of this charter is not how the holistic release of user-facing products is done? I feel there is additional value to be added on top of vetted and effective end-to-end solutions that are built by components in an end-to-end way. I would imagine the packaging, the curation, and distribution to be the focus of this group and is certainly a place for value. I would have been interested in the description of the group to be more focused on these kinds of activities. With that framing, this would make more sense but under the current framing, many of the tasks here should be shared by all teams (like making data flow more efficient).

Reply:
I mentioned the 'scope' in another comment, but will reiterate here the value added to have a group specifically facilitating component-to-component interactions and with the bandwidth to put everything in the context of the whole DCP.

In terms of products by specific components, I think the connection to the user is really where DataOps adds value. DataOps can serve as a user-facing group representing the whole of the DCP. Each component may have some user-facing aspect, but lack that as a _focus_ and they might struggle to put their products in the context of the whole of the DCP. Users are component-agnostic so there should be a user-facing group that is also component-agnostic.

Reply:
@TimothyTickle does this answer your questions, or is there still more to resolve here?

Reply:
I think that this has to be carefully done to make sure POs are not blocked from users, our UX team has a healthy program, and everyone keeps their interests on DCP as a product. That being said I have expressed my concern and this should not be blocking.